webhook/webhookworker.py

```diff
@@ -458,6 +458,32 @@ def __prepare_gyms_data(self, gym_data):
 
         return ret
 
+    def __prepare_pokestops_data(self, pokestop_data):
+        ret = []
+
+        for pokestop in pokestop_data:
+            if self.__is_in_excluded_area([pokestop["latitude"], pokestop["longitude"]]):
+                continue
+
+            pokestop_payload = {
+                "pokestop_id": pokestop["pokestop_id"],
```

Comment:
I know PA doesn't (currently) support it, but add pokesotp name and image url here as well.

They would add so much worth.

Reply:
Its already included, I also opened a PR for PA to support it.

Reply:
🤦‍♂ yeah I see that you did that...